stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/DeletedMessageListItemPredicate.kt

```diff
@@ -22,6 +22,15 @@ import io.getstream.chat.android.client.ChatClient
 /**
  * Predicate class used to filter [MessageListItem.MessageItem] items which are deleted. Used by [MessageListView.setDeletedMessageListItemPredicate].
  */
+@Deprecated(
+    message = "Deprecated in favor of filtering in the ViewModel by using " +
+        "'io.getstream.chat.android.common.state.DeletedMessageVisibility'" +
+        " together with 'com.getstream.sdk.chat.viewmodel.messages.setDeletedMessageVisibility()'",
+    replaceWith = ReplaceWith(
+        "io.getstream.chat.android.common.state.DeletedMessageVisibility"
+    ),
```

Comment:
I think the goal of ReplaceWith to auto replace `DeletedMessageListItemPredicate` text with `io.getstream.chat.android.common.state.DeletedMessageVisibility`. Don't think this is enough to fix somebody's code.

Reply:
Agreed, in most cases it will not fix someone's code.

It's here mostly so that if someone is using the predicate apart from the setter method in `MessageListView` for whatever purpose, they can more easily locate `DeletedMessageVisibility`.

But I think your point still stands.